control_software/scripts/hera_catcher_up.py

```diff
@@ -67,17 +67,6 @@ def run_on_hosts(hosts, cmd, user=None, wait=True):
 cpu_mask = '0x0004'
 run_on_hosts([args.host], ['taskset', cpu_mask, 'hashpipe_redis_gateway.rb', '-g', args.host, '-i', '0'])
 
-# Wait for the gateways to come up
-time.sleep(15)
-
-# Upload config file location
-# NOTE: This has to come before template generation!
-# Generate the BDA config file and upload to redis
-if not args.nobda:
-    print('Create configuration file')
-    run_on_hosts([args.host], python_source_cmd + bda_config_cmd + ['-c','-r', '/tmp/bdaconfig.txt'], wait=True)
-    os.system('scp "%s:%s" "%s" ' % ('hera-sn1', '/tmp/bdaconfig.txt','/tmp/bdaconfig.txt') )
-    
 time.sleep(10)
```

Comment:
Is this sleep still necessary?

Reply:
I tried to run a1759f0 (where this sleep was removed), and it didn't work. I added it back in with 2b1fdd7 and that worked on site.